client/components/confirmation-modal/styles.scss

```diff
@@ -12,8 +12,8 @@
 	}
 
 	.components-modal__header {
-		margin: 0 -#{$grid-unit-30} $grid-unit-30;
```

Comment:
It seems that you need to fix a spacing issue? A friendly reminder that we should test that we are not affecting other usages of the component

Reply:
I think this is change from other PR I'm working on that has crept into this one. Thanks for pointing out, I'll remove it from here